### DISCUSSION TITLE
[ feature request ] update / replace an asset instead of uploading

### DISCUSSION THREAD

**Discussion by raidolo** [simulot/immich-go#501] [Ideas]
Hi @simulot 

Currently the only command supported is to upload an asset to immich server, but I've seen there are APIs to update an asset or to replace it.
It would be great if you can include these commands in immich-go. 
These commands are not supported even by immich-cli at the moment, but it could be very handy to fix errors and to replace assets without the need to delete them first from the library.

Thanks.

**Comment by simulot** [comment]
You're right. Once uploaded, immich-go ignore files.

I can imagine several means to update immich assets using immich-go:

1. reprocess the same source, with different parameters
2. update immich assets with a data set
    - a GPS track
    - a google map history


The 1. would be handy to reprocess a data set with a new version of immich-go
